frontend/src/scenes/insights/PersonModal.js

```diff
@@ -33,7 +34,14 @@ export function PersonModal({ visible, view }) {
                 <div
                     style={{ marginBottom: 16, display: 'flex', alignItems: 'center', justifyContent: 'space-between' }}
                 >
-                    Found {people.count === 99 ? '99+' : people.count} {people.count === 1 ? 'user' : 'users'}
+                    <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-start' }}>
+                        Found {people.count === 99 ? '99+' : people.count} {people.count === 1 ? 'user' : 'users'}
+                        {(view === ViewType.TRENDS || view === ViewType.STICKINESS) && (
```

Comment:
WDYT about putting this behind a feature flag?

Advantage: We can turn it off more easily if/as we discover bugs
Disadvantage: More management work, won't work well with self-hosted.